i18n/en.json

```diff
@@ -57,5 +57,7 @@
 	"transkribus-init-process-error": "Failed to initialize Transkribus process",
 	"transkribus-failed-process-error": "Transkribus process failed",
     "transkribus-no-lang-error": "No language was selected",
-    "transkribus-multiple-lang-error": "Multiple languages are not allowed, specify one language"
+    "transkribus-multiple-lang-error": "Multiple languages are not allowed, specify one language",
+    "transkribus-browse-public-models": "Browse all public models for Transkribus",
```

Comment:
I know we've T330061 open to sort this out, but I think that having it say "public models" here when the field above is "Languages" might be confusing. Users wouldn't know what a model is at this point. Maybe just say "public language models" here; what do you think? @satdeepgill what do you think?

Reply:
I agree! “public language models” would be better understood.